css/navigation.css

```diff
@@ -0,0 +1,7 @@
+.nav-container {
+  height: auto;
```

Comment:
by default height is auto, is there any other usage of **height: auto**, rest of the code looks clean.

good to merge

Reply:
If I do not add height: auto , it takes excessive space of the parent, which I don't want.
Thanks for your input